Composer/packages/client/src/recoilModel/dispatchers/multilang.ts

```diff
@@ -59,7 +60,26 @@ export const multilangDispatcher = () => {
   const setLocale = useRecoilCallback(
     ({ set, snapshot }: CallbackInterface) => async (locale: string, projectId: string) => {
       const botName = await snapshot.getPromise(botDisplayNameState(projectId));
-
+      const botProjectsMetaData = await snapshot.getPromise(botProjectSpaceSelector);
+      const rootBotProjectId = await snapshot.getPromise(rootBotProjectIdSelector);
+      if (projectId === rootBotProjectId) {
+        for (let i = 0; i < botProjectsMetaData.length; i++) {
+          if (!botProjectsMetaData[i].isRootBot && !botProjectsMetaData[i].isRemote) {
+            const skillBotProjectId = botProjectsMetaData[i].projectId;
+            const settings = await snapshot.getPromise(settingsState(skillBotProjectId));
+            const languages = settings.languages;
+            const defaultLang = settings.defaultLang;
```

Comment:
Should we pick the active locale of the root bot instead of default locale?

Reply:
From current code logic, I think a new language is copied from the default language.